dotcom-rendering/src/server/dev-index.html

```diff
@@ -309,6 +309,40 @@ <h2>Test Articles By Element (Missing)</h2>
                 },
             ];
 
+			const urls = new Set();
+
+			const checkClipboard = (e) => {
+				if (document.visibilityState !== "visible") return;
+				if (!navigator?.clipboard) return;
+				if (!document.hasFocus()) return;
+
+				navigator.clipboard.readText().then(text => {
+					const guurl = "https://www.theguardian.com/";
```

Comment:
Could we make this an array of` ["https://www.theguardian.com/", "https://m.code.dev-theguardian.com/", "localhost:9000/"]`?

Reply:
```suggestion
					const guurl = ["https://www.theguardian.com/", "https://m.code.dev-theguardian.com/", "http://localhost:9000/"].find(base => text.startWith(base));
```